test/specs/scenarios/document.js

```diff
@@ -201,8 +201,8 @@ export const documentScenarios = async (lang) => {
         confirm.verifyUseAnotherFileError(copy)
       })
 
-      it('should return glare detected message on front and back of doc', async () => {
-        driver.get(`${baseUrl}&async=false&useUploader=true`)
+      it.skip('should return image quality message on front of doc', async () => {
```

Comment:
These tests are skipped because the production APIs haven't been fully enrolled with `image-quality` service.

Reply:
Might be better to include a note reminder to re-enable in the code base itself.